drivers/SmartThings/matter-pump/profiles/pump-level.yml

```diff
@@ -0,0 +1,21 @@
+name: pump-level
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+  - id: pumpOperationMode
+    version: 1
+  - id: pumpControlMode
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Irrigation
+metadata:

```

Comment:
@HunsupJung why is pump using a custom presentation?

Comment copied from here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1422#discussion_r1632061902

Reply:
It's related naming of switchLevel capability. switchLevel Capability is displayed as `dimmer`. so we needed to modify it to `level` through device configuration.

Reply:
I see. Seems like that's the best option for now but let's talk internally about longer term options so this isn't necessary

Reply:
@tpmanley 
Yes, please tell me if there is any part that needs to be proceeded.

Reply:
Let's plan to wrap up discussion on this topic this week so we can get this driver to beta next week.

Reply:
I think we will need to move forward with the metadata for now to modify the presentation to say `level`